backend.py

```diff
@@ -2,41 +2,62 @@
 from flaskext.mysql import MySQL
 from datetime import datetime
 import logic
+import json
+from collections import defaultdict
 app = Flask(__name__)
 
-app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
-app.config['MYSQL_USER'] = 'Osama'
-app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
+# app.config['MYSQL_USER'] = 'Osama'
+# app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_DATABASE_DB'] = 'wordsense'
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_DATABASE_USER'] = 'pnkls'
+app.config['MYSQL_DATABASE_PASSWORD'] = 'root'
 app.config['MYSQL_DATABASE_DB'] = 'wordsense'
 
 mysql = MySQL(app)
 conn = mysql.connect()
 cursor = conn.cursor()
 
+def GetIndexing():
+    f = open("indexing.txt", "r")
+    dictionary = json.loads(f.read())
+    return dictionary
+
 @app.route('/', methods =["GET", "POST"])
 def homepage():
+    # Get the indexing in memory so that we have it until the application is closed
+    dictionary = GetIndexing()
```

Comment:
This is a good idea, so then we don't have to make calls everytime to get the index